Tests/AppTests/ErrorPageModelTests.swift

```diff
@@ -0,0 +1,41 @@
+@testable import App
+
+import XCTVapor
+
+class ErrorPageModelTests: AppTestCase {
+
+    func test_404() throws {
+        // setup
+        let status = HTTPResponseStatus(statusCode: 404)
+
+        // MUT
+        let model = ErrorPage.Model(status: status, error: nil)
+
+        // validate
+        XCTAssertEqual(model.errorMessage, "404 - Not Found")
+    }
+
+    func test_500_without_error() throws {
+        // setup
+        let status = HTTPResponseStatus(statusCode: 500)
+
+        // MUT
+        let model = ErrorPage.Model(status: status, error: nil)
+
+        // validate
+        XCTAssertEqual(model.errorMessage, "500 - Internal Server Error")
+    }
+
+    func test_500_with_error() throws {
+        // setup
+        let status = HTTPResponseStatus(statusCode: 500)
+        let error: AbortError = nil // I need an AbortError here but don't want to stand up the whole stack just to test the view model.
```

Comment:
I really don't want to set up the whole stack here, but I do need to throw (or somehow get hold of) an `AbortError`. Any ideas @finestructure?

Reply:
There’s a function `Abort()` that you should be able to call (but I’m not quite sure without a computer to hand if it returns an `AbortError` or throws)